data/kadi4mat_json/atmospheric_corrosion_models/corrosion-degradation-model-including-coating-effe.json

```
@@ -0,0 +1,132 @@
+  "extras": [
+    {
+      "key": "Parameters",
+      "type": "dict",
+      "value": [
+        {
+          "key": "Coating Life",
+          "type": "float",
+          "unit": "years",
+          "value": null
+        },
+        {
+          "key": "Transition Time",
+          "type": "float",
+          "unit": "years",
+          "value": null
+        },
+        {
+          "key": "Long term thickness of corrosion wastage",
+          "type": "float",
+          "unit": "mm",
+          "value": null
+        }
+      ]
+    },
+    {
+      "key": "Formula",
+      "type": "dict",
+      "value": [
+        {
+          "key": "Material Loss",
+          "type": "str",
+          "value": "d(t)=d_{\\infty}\\left(1-\\mathrm{e}^{-\\left(t-\\tau_c\\right) / \\tau_t}\\right), \\quad t>\\tau_c"
+        },
+        {
+          "key": "Material Loss t<\\tau_c",
+          "type": "str",
+          "value": "d(t)=0, \\quad t \\leqslant \\tau_c"
+        }
+      ]
+    }
+  ],
+  "files": [],
+  "id": 227,
+  "identifier": "corrosion-degradation-model-including-coating-effe",
+  "last_modified": "2024-08-15T08:01:55.834569+00:00",
+  "license": null,
+  "links": [
+    {
+      "created_at": "2024-08-15T07:59:38.109998+00:00",
+      "creator": {
+        "displayname": "aravinth_ravikumar",
+        "id": 5,
+        "orcid": null
+      },
+      "id": 188,
+      "last_modified": "2024-08-15T07:59:38.110006+00:00",
+      "name": "publication",
+      "record_to": {
+        "created_at": "2024-07-29T13:27:18.015525+00:00",
+        "creator": {
+          "displayname": "aravinth_ravikumar",
+          "id": 5,
+          "orcid": null
+        },
+        "description": "",
+        "extras": [
+          {
+            "description": "Date of the creation of document. Can be identical to the date of the creation of the record if no better date is known.",
+            "key": "creationDate",
+            "type": "date",
+            "validation": {
+              "required": true
+            },
+            "value": "2024-07-29T13:27:18.056381+00:00"
+          },
+          {
+            "description": "Language of the document.",
+            "key": "language",
+            "type": "str",
+            "validation": {
+              "required": true
+            },
+            "value": "english"
+          },
+          {
+            "key": "publicationDate",
+            "type": "date",
+            "validation": {
+              "required": true
+            },
+            "value": "1999-01-01T00:00:00+00:00"
+          },
+          {
+            "key": "pages",
+            "type": "str",
+            "value": "425--445"
+          },
+          {
+            "key": "journalName",
+            "type": "str",
+            "value": "Marine structures"
+          }
+        ],
+        "id": 197,
+        "identifier": "reliability_of_maintained_corrosion_protected_plat",
```

```suggestion
        "identifier": "Soares1999",
```
Lets use german citation style identifiers for publication identifiers so First Author Lastname + Year + {A,B,C...) if multiple papers in one year.

---

Has this been fixed?

---

yes